### DISCUSSION TITLE
How is this project connected to tabulapdf/tabula?

### DISCUSSION THREAD

**Discussion by jbarlow83** [chezou/tabula-py#375] [Q&A]
Is this is a fork of
https://github.com/tabulapdf/tabula
or is there any other connection?

**Comment by chezou** [comment]
As I described in the README, it is a wrapper of tabula-java, which is the core of the tabula app (tabulapdf/tabula).

> tabula-py is a simple Python wrapper of [tabula-java](https://github.com/tabulapdf/tabula-java), which can read tables in a PDF. 

Tabula app is a web UI on top of tabula-java. So, tabula-py isn't a fork of tabula app itself.
